src/spatialdata/_utils.py

```diff
@@ -156,7 +156,11 @@ def _compute_paddings(data: DataArray, axis: str) -> tuple[int, int]:
 def multiscale_spatial_image_from_data_tree(data_tree: DataTree) -> MultiscaleSpatialImage:
     d = {}
     for k, dt in data_tree.items():
-        v = dt.values()
+        try:
+            v = dt.values()
+        except TypeError as e:
+            pass
+            raise e
```

Comment:
What is the purpose of this? Is it just re-raising the same error?

Reply:
I forgot it there while debugging, I will remove it.